packages/tooltip/src/Tooltip.story.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import { Align, Justify } from '@leafygreen-ui/popover';
+import Tooltip, { TriggerEvent, Variant } from '.';
+
+function ControlledTooltip() {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Tooltip
+      open={open}
+      setOpen={setOpen}
```

Comment:
I might be missing something, but the button isn't actually firing the `setOpen` event.

Reply:
yeah the button is, but pass it to Tooltip so we can handle closing on escape and backdrop clicks :)